spec/requests/api/v0/events_request_spec.rb

```diff
@@ -21,12 +23,51 @@
       }
     end
 
+    let(:attributes_short) do
+      {
+        'events': [
+          {
+            'data': data,
+            'type': 'tutor quiz',
```

Comment:
Hmm, are we planning on doing something w/ 'type'? I see we're calling
KafaClient.produce directly w/ the event data and topic, but ignoring the type.

Reply:
@reedstrm i'm not sure yet how exactly we'll use type..my thought is that we'll adopt a convention where this translates to a key to look up in the schema registry.  putting here for now, i expect we'll refine it.

Reply:
Right same problem as user_uuid - we've been thinking we want stuff wrapping the actual event data. Maybe we _store_ the top level dict, rather than the data-as-sent.

Reply:
here..the type is passed in (i think) to determine the schema registry key that's used to validate the data.  i dont think we're planning to send it in thru the data.  but you prob. know this better than i do :)